cosmopolitan_continents/migrations/0002_auto_20160124_1705.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cosmopolitan_countries', '0001_initial'),
+        ('cosmopolitan_continents', '0001_initial'),
+        ('cosmopolitan_currencies', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='cosmopolitancontinent',
+            name='countries',
+            field=models.ManyToManyField(to='cosmopolitan_countries.CosmopolitanCountry', related_name='related_continent_country'),
```

Comment:
I think we should reduce all the naming spacing of "cosmopolitan". Why not import from an app called cosmopolitan, and then we can also get rid of `Cosmopolitan*` in the model names too I guess.

`to='cosmopolitan.Country'`